reana_client/cli/quotas.py

```diff
@@ -99,22 +93,11 @@ def quota_show(
             )
             sys.exit(1)
         if not report:
-            usage = quota[resource].get("usage")
-            limit = quota[resource].get("limit")
-            limit_str = ""
-            percentage = ""
-            kwargs = dict()
-            if limit and limit.get("raw", 0) > 0:
-                health = quota[resource].get("health")
-                percentage = usage_percentage(usage.get("raw"), limit.get("raw"))
-                limit_str = f"out of {limit.get(human_readable_or_raw)} used"
-                kwargs = dict(fg=REANA_RESOURCE_HEALTH_COLORS.get(health))
-            else:
-                limit_str = "used"
-
-            return click.secho(
-                f"{usage[human_readable_or_raw]} {limit_str} {percentage}", **kwargs
+            resource_usage, health = get_quota_resource_usage(
+                quota[resource], human_readable_or_raw
             )
+            kwargs = dict(fg=REANA_RESOURCE_HEALTH_COLORS.get(health))
+            return click.secho(resource_usage, **kwargs)
```

Comment:
*suggestion(non-blocking):* I would just make it as one line without using `kwargs`.

```python
return click.secho(resource_usage, fg=REANA_RESOURCE_HEALTH_COLORS.get(health))
```

Or

```python
health_message_color = REANA_RESOURCE_HEALTH_COLORS.get(health)
return click.secho(resource_usage, fg=health_message_color)
```